src/styles/components/ec-card/ec-card.css

```diff
@@ -0,0 +1,38 @@
+:root {
+  --ec-card-interactive-transition-duration: 0.4s;
+  --ec-card-interactive-transition-timing-function: ease;
+}
+
+.ec-card {
+  @apply tw-rounded tw-p-8 tw-bg-light tw-shadow-level-1;
+
+  word-break: break-word;
```

Comment:
Can we use `.tw-break-words` here? I know it's not doing exactly the same thing, it does: `word-wrap: break-word`. Can we check if the result is still the same: https://tympanus.net/codrops/css_reference/word-wrap/?

Reply:
True, it seems we can use it, thx!